c-bindings/c-bindings.go

```diff
@@ -25,6 +25,32 @@ type jsonValue struct {
 	owned []*C.struct_JsonnetJsonValue
 }
 
+type importer struct {
+	cb  *C.JsonnetImportCallback
+	ctx unsafe.Pointer
+}
+
+// Import fetches data from a given path by using c.JsonnetImportCallback
+func (i *importer) Import(importedFrom, importedPath string) (contents jsonnet.Contents, foundAt string, err error) {
+	var (
+		success   = C.int(0)
+		foundHereC *C.char
+	)
+
+	resultC := C.jsonnet_internal_execute_import(i.cb, i.ctx, C.CString(importedFrom), C.CString(importedPath), &foundHereC, &success)
```

Comment:
I think `C.CString(importedFrom)` is not right here. There is a slight difference in the C and Go API. The importer in Go gets the full path of the file as `importedFrom` while in C API it's just the directory it is in (the Go approach here is supposed to be better and less opinionated, but C API should be 100% compatible). Directory is understood here as *the part of the path before the last slash*. 

You can take a look at how it's handled in C++ code: https://github.com/google/jsonnet/blob/master/core/vm.cpp#L761 for how the imports are handled in C++.